setup.py

```diff
@@ -32,20 +30,19 @@
     classifiers=[
         "Environment :: Web Environment",
         "Framework :: Django",
-        "Framework :: Django :: 2.2",
-        "Framework :: Django :: 3.0",
-        "Framework :: Django :: 3.1",
         "Framework :: Django :: 3.2",
+        "Framework :: Django :: 4.0"
+        "Framework :: Django :: 4.1"
+        "Framework :: Django :: 4.2"
```

Comment:
```suggestion
        "Framework :: Django :: 4.0",
        "Framework :: Django :: 4.1",
        "Framework :: Django :: 4.2",
```
This needs to be changed or PyPi will (╯°□°）╯︵ ┻━┻

Reply:
Whopsie! :sweat_smile: